spec/factories/course_assessment_submissions.rb

```diff
@@ -72,6 +72,37 @@
       end
     end
 
+    trait :with_past_answers do
+      after(:build) do |submission|
```

Comment:
See if the existing flow for re-attempting questions can be used to set up answers without having to manually modify the `current_answer` flag.